### PR TITLE
update project

### PR DESCRIPTION
**Pull request by ktalanda**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,9 +25,9 @@ proguard/
 
 # Log Files
 *.log
+
+#IntelliJ files
 .idea/
+*.iml
 
-/local.properties
```
Comment (ktalanda) [inline .gitignore:30]: already covered by `.idea/`

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#Mon Dec 28 10:00:20 PST 2015
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
```
Comment (ktalanda) [inline gradle/wrapper/gradle-wrapper.properties:6]: those files are not excluded by `.gitignore` so should be added to repo
+ it indicates current gradle version

File: .gitignore (hunk 1 above)
Comment (polok) [reply .gitignore:30]: @ktalanda I would leave local.properties file

File: gradlew [hunk 3]
```diff
@@ -42,11 +42,6 @@ case "`uname`" in
     ;;
 esac
 
-# For Cygwin, ensure paths are in UNIX format before anything is touched.
```
Comment (ktalanda) [inline gradlew:45]: those changes were generated after gradle-2.14.1 was applied

File: .gitignore (hunk 1 above)
Comment (polok) [reply .gitignore:30]: Ah, it's there 👍

**Review by polok — APPROVED** [review]

**Pull request merged**